t/python/t0010-job.py

```diff
@@ -672,6 +672,40 @@ def cancel_on_start(future, jobid):
         # synchronous job.result() test
         self.assertEqual(job.result(self.fh, ids[3]), result[ids[3]].get_info())
 
+    def test_33_get_job(self):
+        self.sleep_jobspec = JobspecV1.from_command(["sleep", "5"])
+        jobid = job.submit(self.fh, self.sleep_jobspec)
+        meta = job.get_job(self.fh, jobid)
+        assert isinstance(meta, dict)
+        for key in ['id',
+                    'userid',
+                    'urgency',
+                    'priority',
+                    't_submit',
+                    't_depend',
+                    'state',
+                    'name',
+                    'ntasks',
+                    'ncores',
+                    'duration',
+                    'nnodes',
+                    'result',
+                    'runtime',
+                    'returncode',
+                    'waitstatus',
+                    'nodelist',
+                    'exception']:
+            assert key in meta 
+
+        assert meta['id'] == jobid
+        assert meta['name'] == "sleep"
+        assert meta['state'] in ["SCHED", "DEPEND", "RUN"]
+        assert meta['ntasks'] == 1
+        assert meta['ncores'] == 1
```

Comment:
We don't use pytest so I think these should be unittest assertions like ``self.assertEqual(meta['ncores'], 1)``